manifest-rhel-8.6.yaml

```diff
@@ -157,7 +157,7 @@ packages:
 # Packages pinned to specific repos in RHCOS
 repo-packages:
   # we always want the kernel from BaseOS
```

Comment:
This comment is out of date now, and should probably be changed to reference the reason why we're doing this.

Reply:
Should've updated this before re-testing but we will be switching back to BaseOS in the next few weeks.
Not ideal but let's ignore this due to time constraints.